data/templates/pattern_base.html

```diff
@@ -0,0 +1,27 @@
+{%- set bootstrap_version = '3.0.3' -%}
```

Comment:
This was used in another template for loading bootstrap from a CDN, and appears to be no longer needed.